src/jit/lir.cpp

```diff
@@ -1510,7 +1510,7 @@ LIR::Range LIR::EmptyRange()
 // static
 LIR::Range LIR::SeqTree(Compiler* compiler, GenTree* tree)
 {
-    // TODO(pdg): it would be great to assert that the node has not already been
+    // TODO-LIR: it would be great to assert that the tree has not already been
     // threaded into an order, but I'm not sure that will be practical at this
```

Comment:
Do we plan to fix this in the near term? If not, I'd just remove the "TODO-LIR", but leave the comment.